docs/api/instruments/jobinyvon/index.rst

```diff
@@ -0,0 +1,13 @@
+.. module:: pymeasure.instruments.jobinyvon
+
+######
+JobinYvon
+######
```

Comment:
You can add spaces in the title as it is meant for humans. Also, the hashes should be as long as the text, for example:
```suggestion
##########
Jobin Yvon
##########
```